pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+)
+
+type accessTokenOptions struct {
+	namespace      string
+	serviceAccount string
+	roleBinding    string
+}
+
+var accessTokenOpts *accessTokenOptions
+
+var accessTokenExamples = strings.Trim(`
+Create an access token in a Kubernetes cluster for one or more member clusters, if the Service Account or RoleBinding does not exit, antctl will create one
```

Comment:
Could you check and update all "in a Kubernetes" in the PR? Again, it is too generic to understand. Please use "leader cluster" or "member cluster".

Reply:
We will get manifests from the antrea release url, such as `https://github.com/antrea-io/antrea/releases/download/v1.5.1/antrea-multicluster-leader-global.yml`. When we use dirty versions, we'll get a url `https://github.com/antrea-io/antrea/releases/download/v1.6.0-dev-f619dbd9.dirty/antrea-multicluster-leader-global.yml`, then we'll not get the manifests cause the v1.6.0-dev-f619dbd9.dirty is not released.

Reply:
I see. In that case, we can check the version, and get the yaml from main if the version is not a released one?

BTW, we should provide an option to use a downloaded file too.

Reply:
Thanks for the comments. I think it's okay to get the latest manifest as the default manifest. I'll edit my code and push again.

Reply:
Done. Now if the version not set, antctl will use the latest version from main branch.